dashboard/src/components/Filter/CheckboxSection.tsx

```diff
@@ -48,6 +48,12 @@ export interface ISectionItem {
   isGlobal?: boolean;
 }
 
+const extractCheckboxValue = (value: string): string => {
```

Comment:
string | undefined in case the array access doesn't exist

Reply:
Also, you can do something like this `type SplitString = `${string}__${string}`;`  to have a more exact type

Reply:
NICE TIP!!!!!!!